functions.php

```diff
@@ -129,12 +132,40 @@ function ampconf_widgets_init() {
 }
 add_action( 'widgets_init', 'ampconf_widgets_init' );
 
+/**
+ * Filters the AMP custom css to include the CSS from out theme.
+ *
+ * @param string $css The combined custom css.
+ *
+ * @return string $css
+ */
+function ampconf_custom_styles( $css ) {
+	$path   = get_template_directory() . '/assets/dist/css/main.css';
+	$styles = file_get_contents( $path ); // phpcs:ignore WordPress.WP.AlternativeFunctions -- Not a remote file.
+
+	return $styles . $css;
+}
+add_filter( 'amp_custom_styles', 'ampconf_custom_styles' );
```

Comment:
This can be eliminated in favor of regular enqueueing with https://github.com/Automattic/amp-wp/pull/887

Reply:
Styles are now enqueued the standard method.